Kryptonite/Kryptonite.xcdatamodeld/krSSH.xcdatamodel/contents

```diff
@@ -1,13 +1,36 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="12141" systemVersion="16E195" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="12141" systemVersion="16F73" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
+    <entity name="CommitSignatureLog" syncable="YES">
+        <attribute name="author" optional="YES" attributeType="String" syncable="YES"/>
```

Comment:
I'm not sure this matters since we're not syncing the logs right now anyways. It seems to be on by default. In fact, all the attributes set syncable to yes.